### ISSUE TITLE
is the cluster consquence visible

### ISSUE THREAD

**Issue by shichenhu** [XuyangAbert/EUFSFC#1]
this object is a brilliant idea, but is this possible that we can see the consquence more audio-visual,that it can be seen in graphical

**Comment by XuyangAbert** [comment]
Hi, @shichenhu, thanks for your interest in this project. If I understand correctly, you would like to visualize the feature clustering results as scatter plot, right?

**Comment by shichenhu** [comment]
exactly，and after i used it in my project , i found the difintion

**Comment by shichenhu** [comment]
exactly, i think it would be more credible if we can see the result as a plot, like the Fcluster’  correlation with other feature in the same class, but i don't have an idea about how to visualize it yet

**Comment by shichenhu** [comment]
and i use it in my project, maybe it's because the uniqueness of my feature characteristics, the Fcluster i choose is the feature that has a little connection with many feature, but not to reduce the most redundant feature, do you think it's possible if i modify the difinition of fitness,not to sum up all the feature , but to select the highest 5 DisC .Do you think this would make this would make the algorithm more suitable for my project

**Comment by shichenhu** [comment]
again ,thank you for sharing your code with us, it really helps a lot, and help me to understand your article, it really represent the spirit of open Internet, after digging for days of finding the most suitable way to solve my problem, i'm so thrilles to see this generous sharing

**Comment by XuyangAbert** [comment]
> exactly, i think it would be more credible if we can see the result as a plot, like the Fcluster’ correlation with other feature in the same class, but i don't have an idea about how to visualize it yet

sure, let me spend some time during the weekend to think and implement this feature. thanks for your suggestions.

**Comment by XuyangAbert** [comment]
> and i use it in my project, maybe it's because the uniqueness of my feature characteristics, the Fcluster i choose is the feature that has a little connection with many feature, but not to reduce the most redundant feature, do you think it's possible if i modify the difinition of fitness,not to sum up all the feature , but to select the highest 5 DisC .Do you think this would make this would make the algorithm more suitable for my project

It seems that the problem may come from the degree of similarity/redundancy among your features. You can try to modify the fitness function or tune the "neirad" variable in the "pseduo_peaks" function . BTW, are your features continuous or categorical variables?

**Comment by XuyangAbert** [comment]
> again ,thank you for sharing your code with us, it really helps a lot, and help me to understand your article, it really represent the spirit of open Internet, after digging for days of finding the most suitable way to solve my problem, i'm so thrilles to see this generous sharing

no problem and thanks for your suggestions again. In case you have further questions, you can reach me out to my email "xyan@aggies.ncat.edu".
